cores/arduino/HardwareTimer.cpp

```diff
@@ -850,8 +850,9 @@ void HardwareTimer::setInterruptPriority(uint32_t preemptPriority, uint32_t subP
   * @param  callback: interrupt callback
   * @retval None
```

Comment:
Can you add a comment about the new parameter (like it is done for other parameter)